src/routes/builder/card/+page.svelte

```
@@ -98,19 +98,19 @@
   <div class="my-4 flex flex-wrap space-x-4">
     <Label class="mb-4 w-full font-bold">Size</Label>
     {#each sizes as size}
-      <Radio class="my-1 w-16" name="interactive_card_size" bind:group={cardSize} value={size}>{size}</Radio>
+      <Radio class="my-1" classes={{ label: "w-16" }} name="interactive_card_size" bind:group={cardSize} value={size}>{size}</Radio>
     {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)